fluentd/lib/fluent-plugin-viaq_data_model/lib/fluent/plugin/viaq_data_model_log_level_normalizer.rb

```diff
@@ -0,0 +1,98 @@
+#
+# Fluentd ViaQ data model Filter Plugin
+#
+# Copyright 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+module ViaqDataModel
+# https://github.com/ViaQ/elasticsearch-templates/blob/master/namespaces/_default_.yml#L63
+    NORMAL_LEVELS = {
```

Comment:
This could be a Set rather than an identity Hash. Not a big deal, it's just surprising to see a Hash with value == key for all entries.

Reply:
this is just a carry over from what already existed